new_platforms/opteesim/Trusted/cyres_sim.c

```diff
@@ -229,14 +233,30 @@ static void gen_cert(
             break;
     }
 
-    DERBuilderContext der_ctx = {0};
-    uint8_t der_buffer[DER_MAX_SIZE] = {0};
-    DERInitContext(&der_ctx, der_buffer, DER_MAX_SIZE);
-
-    cert_fp(&der_ctx, id, signer, code_authority, tbs);
-
-    id->cert_size = sizeof(id->cert);
-    oe_assert(DERtoPEM(&der_ctx, R_CERT_TYPE, id->cert, &id->cert_size) == 0);
+    {
+        DERBuilderContext der_ctx = {0};
+        uint8_t der_buffer[DER_MAX_SIZE] = {0};
+        DERInitContext(&der_ctx, der_buffer, DER_MAX_SIZE);
+
+        cert_fp(&der_ctx, id, signer, code_authority, tbs);
+
+        {
+            /* Sign the Certificate's TBS region and create the final DER cert
+             */
```

Comment:
single line comments should have the */ at end of comment line

Reply:
I think VS automatically wrapped on the second line to keep the width to the predefined number of characters. Reverted.